### ISSUE TITLE
Remove redundant headings and fix heading levels on 5 of the fundamentals pages

### ISSUE THREAD

**Issue by mcking65** [w3c/wai-aria-practices#56]
Five of the fundamental pages (listed below) have the following problems:

1. There is an H2 with content that is a duplicate of the H1 content
2. All the H3 headings should be H2, H4 headings should be H3, etc.

After #24 is resolved, remove the extraneous H2 that duplicates the H1 and promote the level of subsequent headings within the main content.

This problem is unique to the following 5 of the 6 fundamentals pages. It does not occur elsewhere in the site.
1. Landmark Regions
2. Providing Accessible Names and Descriptions
3. Developing a Keyboard Interface
4. Grid and Table Properties
5. Communicating Value and Limits for Range Widgets

**Comment by mcking65** [comment]
I have reviewed the preview of #62, and this issue is addressed.

**Comment by mcking65** [comment]
Verified still fixed after today's changes to #62.
